app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | Exception:

```

Comment:
Code Style: You are raising exceptions with strings as error messages which is good. But, the error messages could be more descriptive. For example, instead of 'You are not vaccinated', you could have 'The visitor is not vaccinated. Please ensure you are vaccinated before visiting the cafe.' This would provide more context to the user.